examples/sensor_metadata_schema/sensor_metadata_schema.json

```diff
@@ -0,0 +1,84 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Waste Package Monitoring Sensor Metadata",
+    "description": "Metadata for a waste package monitoring sensor, including its unique identifier, position, and measurement capabilities.",
+    "type": "object",
+    "required": [
+        "sensor_id",
+        "position",
+        "measuring_quantity"
+    ],
+    "properties": {
+        "sensor_id": {
+            "description": "A unique identifier for the sensor.",
+            "type": "string"
+        },
+        "package_id": {
+            "description": "A unique identifier for the Waste Package where the sensor is placed.",
+            "type": "string"
+        },
+        "position": {
+            "description": "The position of the sensor defined by location and optional orientation.",
+            "type": "object",
+            "required": [
+                "location"
+            ],
+            "properties": {
+                "location": {
+                    "description": "The XYZ coordinates of the sensor.",
+                    "type": "array",
+                    "items": {
+                        "type": "number"
+                    }
+                },
+                "orientation": {
+                    "description": "The XYZ coordinates of the orientation.",
+                    "type": "array",
+                    "items": {
+                        "type": "number"
+                    }
+                }
+            }
+        },
+        "measuring_quantity": {
```

Comment:
number of entries is missing (dimension), or are we storing displacements with 3 directions as 3 different sensors?

Reply:
Will add something like this:
```json          
    "values": {
        "description": "List of measured values",
        "type": "array",
        "items": {
        "type": "number"
        }
}
```

we may need to keep separate schemas that describe the measurement data structure that comes from the sensors.